tests/nix_tests/test_grpc_only.py

```
@@ -93,15 +106,23 @@
         rsp = grpc_eth_call(api_port, msg)
         ret = rsp["ret"]
         valid = ret is not None
-        if valid and 9002 == int.from_bytes(base64.b64decode(ret.encode()), "big"):
+        if valid and (
+            9002 == int.from_bytes(base64.b64decode(ret.encode()), "big")
```

Ruff

Yoda condition detected

(SIM300)